File: node-jest/src/model/Person.js
/*
 * Copyright (C) 2021 Spanning Cloud Apps. All rights reserved.
 */

class Person {
  constructor(firstName, lastName) {
    this.firstName = firstName;
    this.lastName = lastName;
  }

  getFirstName() {
    return this.firstName;
  }

  getLastName() {
    return this.lastName;
  }
}

module.exports = Person;

File: java/src/main/java/com/example/testcontainers/model/Person.java
/*
 * Copyright (C) 2021 Spanning Cloud Apps. All rights reserved.
 */

package com.example.testcontainers.model;

import lombok.*;

import javax.persistence.*;

@Entity
@Getter
@ToString
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class Person {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "first_name")
    private String firstName;

    @Column(name = "last_name")
    private String lastName;

    public Person(final String firstName, final String lastName) {
        this.firstName = firstName;
        this.lastName = lastName;
    }
}
